crates/ruff_linter/src/rules/flake8_simplify/snapshots/ruff_linter__rules__flake8_simplify__tests__SIM103_SIM103.py.snap

```diff
@@ -101,46 +103,107 @@ SIM103.py:32:9: SIM103 [*] Return the condition `b` directly
 35    |-            return False
    32 |+        return bool(b)
 36 33 | 
-37 34 | 
-38 35 | def f():
+37 34 | def f():
+38 35 |     # SIM103
 
-SIM103.py:57:5: SIM103 [*] Return the condition `a` directly
+SIM103.py:39:5: SIM103 [*] Return the condition `a` directly
    |
-55 |   def f():
-56 |       # SIM103 (but not fixable)
-57 |       if a:
+37 |   def f():
+38 |       # SIM103
+39 |       if a:
    |  _____^
-58 | |         return False
-59 | |     else:
-60 | |         return True
+40 | |         return True
+41 | |     return False
+   | |________________^ SIM103
+42 |   
+43 |   def f():
+   |
+   = help: Replace with `return bool(a)`
+
+ℹ Unsafe fix
+36 36 | 
+37 37 | def f():
+38 38 |     # SIM103
+39    |-    if a:
+40    |-        return True
+41    |-    return False
+   39 |+    return bool(a)
+42 40 | 
+43 41 | def f():
+44 42 |     # SIM103
+
+SIM103.py:45:5: SIM103 [*] Return the condition `a` directly
+   |
+43 |   def f():
+44 |       # SIM103
+45 |       if a:
+   |  _____^
+46 | |         return False
+47 | |     return True
+   | |_______________^ SIM103
+48 |   
+49 |   def f():
+   |
+   = help: Replace with `return not a`
+
+ℹ Unsafe fix
+42 42 | 
+43 43 | def f():
+44 44 |     # SIM103
+45    |-    if a:
+46    |-        return False
+47    |-    return True
+   45 |+    return not a
+48 46 | 
+49 47 | def f():
+50 48 |     # OK
+
+SIM103.py:68:5: SIM103 [*] Return the condition `a` directly
+   |
+66 |   def f():
+67 |       # SIM103 (but not fixable)
+68 |       if a:
+   |  _____^
+69 | |         return False
+70 | |     else:
+71 | |         return True
    | |___________________^ SIM103
    |
    = help: Replace with `return not a`
 
 ℹ Unsafe fix
-54 54 | 
-55 55 | def f():
-56 56 |     # SIM103 (but not fixable)
-57    |-    if a:
-58    |-        return False
-59    |-    else:
-60    |-        return True
-   57 |+    return not a
-61 58 | 
-62 59 | 
-63 60 | def f():
+65 65 | 
+66 66 | def f():
+67 67 |     # SIM103 (but not fixable)
+68    |-    if a:
+69    |-        return False
+70    |-    else:
+71    |-        return True
+   68 |+    return not a
+72 69 | 
+73 70 | 
+74 71 | def f():
 
-SIM103.py:83:5: SIM103 Return the condition `a` directly
+SIM103.py:94:5: SIM103 [*] Return the condition `a` directly
    |
-81 |       def bool():
-82 |           return False
-83 |       if a:
+92 |       def bool():
+93 |           return False
+94 |       if a:
    |  _____^
-84 | |         return True
-85 | |     else:
-86 | |         return False
+95 | |         return True
+96 | |     else:
+97 | |         return False
    | |____________________^ SIM103
    |
-   = help: Inline condition
-
+   = help: Replace with `return bool(a)`
 
```

Comment:
This is what I'm referencing: when `bool` is _not_ a builtin, this should skip unsafe edits.